### DISCUSSION TITLE
Soft Dependencies

### DISCUSSION THREAD

**Discussion by iGreenDogs** [ScreamingSandals/BedWars#338] [Q&A] [answered]
Hello, in BedwarsPlugin.java on line 64(I think) it has the following lines

@PluginDependencies(platform = PlatformType.BUKKIT, softDependencies = {
        "Vault",
        "Multiverse-Core",
        "Multiworld",
        "MultiWorld",
        "UltimateCore",
        "PlaceholderAPI",
        "BarAPI",
        "PerWorldInventory",
        "SlimeWorldManager",
        "My_Worlds",
        "Parties"
})

Can you please send a link to each one of the plugins so there will be no confusion??

Also, if possible which ones are alike so I don't configure something I do not need twice.

**Comment by zlataovce** [comment, marked as answer]
those are soft dependencies, you don't need those for the plugin to function, BedWars just hooks into those if they are available
